public/app/features/explore/PrometheusListView/ItemLabels.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { RawPrometheusListItemEmptyValue } from '../utils/getRawPrometheusListItemsFromDataFrame';
+
+import { ItemValues } from './ItemValues';
+import { RawListValue } from './RawListItem';
+
+function getMockString(length: number) {
+  let result = '';
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
+}
+
+const value1 = getMockString(8);
+const value2 = getMockString(8);
```

Comment:
randomizing the test input doesn't really add any value in this case and makes the test unnecessarily complex.
if this was intended to try to catch edge-case values we should probably add specific tests for them instead.

can we remove this and just use contsant values?